### PR TITLE
Add com.github.stakira.OpenUtau

### PR DESCRIPTION
**Pull request by orowith2os**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

This needs *just a bit* more work until it's ready. Making a draft so it could be improved in the meantime.
# **AND ADD THIS TO FLATHUB BETA, NOT STABLE**

### CODE REVIEW

File: com.github.stakira.OpenUtau.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  - name: OpenUtau
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/
+      - mv * /app/bin/
+      - chmod +x /app/bin/OpenUtau
+
+    sources:
+      - type: archive #I'd like to build from source, but it doesn't seem to work properly and I can't figure out how to make it build for both aarch64 and x86/
```
Comment (hfiguiere) [inline com.github.stakira.OpenUtau.yml:21]: you don't need to do anything specific to build on aarch64. Unless your build system is broken.

Comment (hfiguiere) [reply com.github.stakira.OpenUtau.yml:21]: I mean **on flathub**

Comment (orowith2os) [reply com.github.stakira.OpenUtau.yml:21]: OpenUtau is built with C#, doesn't dotnet hard-code the arch in the build commands?

Comment (orowith2os) [reply com.github.stakira.OpenUtau.yml:21]: Also, I encountered a dependency hell when I tried building from source, no clue on how that happened, but it did.

Comment (hfiguiere) [reply com.github.stakira.OpenUtau.yml:21]: I don't know what it does, but you can set in the manifest arch dependent build-options and others. `$FLATPAK_ARCH` is set to the right value. Also we do not cross compile either, so if it pick based on the host architecture, then it should be fine.

File: com.github.stakira.OpenUtau.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+app-id: com.github.stakira.OpenUtau
```
Comment (hfiguiere) [inline com.github.stakira.OpenUtau.yml:1]: Should be `io.github`

**Comment by orowith2os** [comment]
Closing this, as I don't have the time to work on nor debug it. The work is still here if anyone wants to pick it up.

**Pull request closed without merging**